backend/src/relation-requests/relation-requests.resolver.ts

```diff
@@ -0,0 +1,69 @@
+import { Resolver, Query, Mutation, Args } from '@nestjs/graphql'
+import { RelationRequestsService } from './relation-requests.service'
+import { RelationRequests } from './entities/relation-requests.entity'
+import { RelationFriend } from '../relation-friend/entities/relation-friend.entity'
+import { RelationRequestsInput } from './dto/create-relation-requests.input'
+import { ValidationPipe } from '@nestjs/common'
+
+@Resolver(() => RelationRequests)
+export class RelationRequestsResolver {
+  constructor(
+    private readonly relationRequestsService: RelationRequestsService
+  ) {}
+
+  //**************************************************//
+  //  MUTATION
+  //**************************************************//
+
+  @Mutation(() => RelationRequests)
+  async createRelationRequests(
+    @Args('data', { type: () => RelationRequestsInput }, ValidationPipe)
+    createRelationRequestsDto: RelationRequestsInput
+  ): Promise<RelationRequests | RelationFriend> {
+    return this.relationRequestsService.create(createRelationRequestsDto)
+  }
+
+  @Mutation(() => RelationRequests)
+  async deleteRelationRequests(
+    @Args('senderId', { type: () => String }) SenderId: string,
+    @Args('ReceiverId', { type: () => String }) ReceiverId: string
+  ): Promise<RelationRequests> {
+    return this.relationRequestsService.delete(SenderId, ReceiverId)
+  }
```

Comment:
your Args name must match the param one, you called SenderId the param and senderId the Args name, just keep using senderId and receiverId without caps